tests/framework/utils/testRandomUtils.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  This Module performs Unit Tests for the randomUtils methods
+"""
+
+#For future compatibility with Python 3
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+
+import os,sys
+import numpy as np
+frameworkDir = os.path.normpath(os.path.join(os.path.dirname(os.path.abspath(sys.argv[0])),os.pardir,os.pardir,os.pardir,'framework'))
+sys.path.append(frameworkDir)
+from utils import utils
+utils.find_crow(frameworkDir)
+from utils import randomUtils
+
+print (randomUtils)
```

Comment:
Necessary? No.  But it does offer an initial check on the module itself, and has been established as precedent by the other unit tests in RAVEN.